dlx_rest/static/js/jmarc.mjs

```diff
@@ -744,6 +778,39 @@ export class Jmarc {
 		)
 	}
 
+	async post_mrk(content) {
```

Comment:
Okay, so digging through this again, I remembered the issue.

When you run jmarc.post(), auth control fields are missing xrefs, since the mrk is externally sourced. Either we have to find a way to provide xrefs for each auth control field, or we have to have a method that doesn't check for them.

Reply:
> Okay, so digging through this again, I remembered the issue.
> 
> When you run jmarc.post(), auth control fields are missing xrefs, since the mrk is externally sourced. Either we have to find a way to provide xrefs for each auth control field, or we have to have a method that doesn't check for them.

Check PR https://github.com/dag-hammarskjold-library/dlx-rest/pull/1378